architecture/infrastructure_components/web_console.adoc

```diff
@@ -38,10 +37,12 @@ link:http://caniuse.com/#feat=websockets[WebSockets].
 .Web Console Request Architecture
 image::web_console_request_arch.png["Web Console Request Architecture"]
 
-The default host name for the web console is whitelisted to communicate with the
-API server. To access the web console using a different host name, you must
+The configured host names and IP addresses for the web console are whitelisted to access the
+API server safely even when the browser would consider the requests to be link:http://www.w3.org/TR/cors/[cross-origin]. To access the API server from a web application using a different host name, you must
```

Comment:
The original issue was that the console being at a different port was considered cross-origin (requests for JSON at API considered a different origin from the web application at a different port). With the web console now at the same origin it doesn't matter as much. But could still be applicable if accessing the web console at a different location from the publicMasterURL for some reason, or for 3rd-party web apps that use our APIs.